### DISCUSSION TITLE
can't play Pop'n Music 8

### DISCUSSION THREAD

**Discussion by hayahayasan** [aldostools/webMAN-MOD#999] [Q&A]
I tried to use HEN to convert "Pop'n Music 8" to PS2 classics format and start it up, but the screen goes black and I can't play it properly. I also checked the location of the config file, but the config file is probably not being applied. Can someone please tell me how to apply the config file on webman mod?

**Comment by hayahayasan** [comment]
The id of popn music 8 is"SLPM-65316".

**Comment by aldostools** [comment]
Try using [PS2 Classics GUI](https://www.psx-place.com/resources/ps2-classics-gui.638/) to apply the CONFIG from https://ps3.aldostools.org/ps2config.html?q=SLPM-65316

**Reply by hayahayasan** [reply]
I tried. but I installed the pkg file and tried to  open, it will turned into  black screen and freezes

**Reply by aldostools** [reply]
Try asking in PSX-PLACE forums.

The CONFIG was posted by @mrjaredbeta and says that it fixes the black screen
https://www.psx-place.com/threads/compatibility-list-ps2-on-ps3.1306/page-279#post-344598

**Reply by hayahayasan** [reply]
I will try the forum. by the way, how do I check whether the config file is applied? I don't know how to check if the config file is applied when mounting.

**Reply by aldostools** [reply]
There is a combo that lets unlock the config mode:
https://www.psx-place.com/threads/ps2-emu-on-ps3-unlock-config-mode-debug-info-mode-menus.13833/

However, in this case -as the game freezes the console- the only way to know if the config was applied is if you don't get the black screen.

**Reply by hayahayasan** [reply]
It doesn't worked. Sorry for not being able to resolve the config issue. I think it's probably not possible, but is there any way to manually apply the config to the iso other than creating a pkg?

**Reply by aldostools** [reply]
The other option is copying the encrypted config to the USRDIR folder of the PS2 Classics Placeholder. The file should be named "CONFIG"

If you're using the non encrypted version of the config, the game will not work

**Reply by hayahayasan** [reply]
Could you please tell me the exact name of the file directory that will contain the config files connected to "USRDIR"? You put the .config file somewhere in the dev_hdd0 file, right?

**Reply by aldostools** [reply]
The full path of the config file in the placeholder is
/dev_hdd0/game/PS2U10000/USRDIR/CONFIG

**Reply by hayahayasan** [reply]
Should I put the "SLPM_653.16.ENC" file in that file location?

**Reply by hayahayasan** [reply]
I tried to put, but id doesn't work. is the file location is right?

Here's a picture where I put.

![unnamed (1)](https://github.com/aldostools/webMAN-MOD/assets/73538253/3ae352ac-13f5-44f2-a332-f3fbe7adc636)
](url)

**Reply by hayahayasan** [reply]
Should I create an "ENC" folder in that location and put the configuration file inside it?

**Reply by aldostools** [reply]
I already told you that the file in the placeholder should be named "CONFIG" (without the quotes)

If you have CFW, mount the unencrypted ISO using webMAN MOD. The non encrypted config should be named like the ISO + .CONFIG. Example: game name.ISO.CONFIG

For HEN, mount the encrypted .BIN.ENC. The encrypted config should be placed in the same directory of the game and named like the .BIN.ENC + .CONFIG. Example: game name.BIN.ENC.CONFIG

**Reply by aldostools** [reply]
Please read the wiki
https://github.com/aldostools/webMAN-MOD/wiki/Game-Paths-&-Covers#ps2-config

**Reply by hayahayasan** [reply]
I can't still load configs. Is the file location is right?
 The original name of the "ISO.BIN.ENC.CONFIG" file is "SLPM_653.16.ENC" taken from websites, but should I use the "SLPM_653.16.CONFIG" file?
![unnamed (1)](https://github.com/aldostools/webMAN-MOD/assets/73538253/0a7dc617-5b0e-4e36-8797-b59b1e9d8db4)

**Reply by hayahayasan** [reply]
I will also try "auto-config" systems.

**Reply by hayahayasan** [reply]
I tried Auto-config, but failed either. I'll show you the picture of file locations later.

**Reply by hayahayasan** [reply]
The "XXXX.BIN.ENC.CONFIG" file was generated after launched ps2 classics launcher. Maybe config file was wrong.

![unnamed (1)](https://github.com/aldostools/webMAN-MOD/assets/73538253/c8a2cc17-c001-436b-ba89-6256d2d8357a)

**Comment by aldostools** [comment]
If auto-config generated the CONFIG, it should work if the game is mounted by webMAN MOD.

If the CONFIG is still not working, I recommend that ask directly to @mrjaredbeta in PSX-PLACE forums.
